sopel/modules/reddit.py

```diff
@@ -113,20 +113,58 @@ def rpost_info(bot, trigger, match):
         return NOLIMIT
 
 
-# If you change this, you'll have to change some other things...
-@commands('redditor')
-@example('.redditor poem_for_your_sprog')
-def redditor_info(bot, trigger, match=None):
+def subreddit_info(bot, trigger, match, commanded=False):
+    """Shows information about the given subreddit"""
+    r = praw.Reddit(
+        user_agent=USER_AGENT,
+        client_id='6EiphT6SSQq7FQ',
+        client_secret=None,
+    )
+    try:
+        r.subreddits.search_by_name(match, exact=True)
+        s = r.subreddit(match)
+        link = "https://www.reddit.com/r/" + s.display_name
+
+        tz = time.get_timezone(bot.db, bot.config, None, trigger.nick,
+                               trigger.sender)
+        time_created = dt.datetime.utcfromtimestamp(s.created_utc)
+        created = time.format_time(bot.db, bot.config, tz, trigger.nick,
+                                   trigger.sender, time_created)
+
+        message = ('[REDDIT] {link}{nsfw} | subscribers ({subscribers}) | Created at {created} | {public_description}')
```

Comment:
Why is it formatted like `subscribers (124732)`?

I would use a thousand separator and something like `124,732 subscribers`

```suggestion
        message = ('[REDDIT] {link}{nsfw} | {subscribers} subscribers | Created at {created} | {public_description}')
```